templates/bkgImages.mustache

```diff
@@ -0,0 +1,15 @@
+{{#bkgImages}}
+  #{{slideId}} {
+    background: url({{{bkgImage}}});
+  }
+{{/bkgImages}}
+
+{{#bkgImages}}
```

Comment:
This is probably unnecessary. You could set it to apply to all slide-class divs instead of this.